File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/inputs/inputWithDefaultCurrency/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.inputs.inputWithDefaultCurrency.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Orders = "orders";

    public static class ORDERS_INPUT_ARGUMENT {
      public static final String Filter = "filter";
    }
  }

  public static class ORDERFILTER {
    public static final String TYPE_NAME = "OrderFilter";

    public static final String Value = "value";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/inputWithDefaultCurrency/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.inputWithDefaultCurrency.expected

import kotlin.String

@Generated
public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  @Generated
  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Orders: String = "orders"

    @Generated
    public object ORDERS_INPUT_ARGUMENT {
      public const val Filter: String = "filter"
    }
  }

  @Generated
  public object ORDERFILTER {
    public const val TYPE_NAME: String = "OrderFilter"

    public const val Value: String = "value"
  }
}
